integrations/quayLogin/cli/init.sh

```diff
@@ -51,9 +51,9 @@ init_scope_configure() {
   _log_msg "Initializing scope configure"
 
   if _is_docker_email_deprecated; then
-    docker_login_cmd=$( docker login -u "$QUAY_USERNAME" -p "$QUAY_PASSWORD" quay.io )
+    docker_login_cmd="docker login -u $QUAY_USERNAME -p $QUAY_PASSWORD quay.io"
```

Comment:
Could you test this with passwords that have special characters like `'`, `"`? We might need to correct this across scripts.